wpe/src/main/cpp/browser/entrypoints.cpp

```diff
@@ -160,6 +160,13 @@ void deleteInputMethodContent(JNIEnv*, jclass, jint pageId, jint offset)
     ALOGV("BrowserGlue::deleteInputMethodContent(%d, %d) [tid %d]", pageId, offset, gettid());
     Browser::getInstance().deleteInputMethodContent(pageId, offset);
 }
+
+void requestExitFullscreenMode(JNIEnv*, jclass, jint pageId)
+{
+    ALOGV("BrowserGlue::requestExitFullscreen(%d) [tid %d]", pageId, gettid());
+    Browser::getInstance().requestExitFullscreen(pageId);
+}
+
```

Comment:
There is an extra black line before the end of the namespace

Reply:
fixed